daemon/delete.go

```diff
@@ -82,7 +85,7 @@ func (daemon *Daemon) DeleteVolumes(volumeIDs map[string]struct{}) {
 }
 
 // Destroy unregisters a container from the daemon and cleanly removes its contents from the filesystem.
-func (daemon *Daemon) Rm(container *Container) error {
+func (daemon *Daemon) Rm(container *Container, forceRemove bool) (err error) {
```

Comment:
Okay, I'm not in favor of this signature. How about helper function and two separate functions `Rm` and `ForceRm` which will call helper with `true` or `false`?